services

```diff
@@ -51,20 +51,8 @@ case "${command}" in
 		stoppingContainers
 		;;
 	"create")
-		echo "Obtaining Mongo DB image"
-		docker pull mongo:3.6
-		echo "Obtaining Latest Orion Image"
-		docker pull fiware/orion
-		echo "Obtaining Latest UltraLight IoT Agent"
-		docker pull fiware/iotagent-ul
-		echo "Obtaining Crate Image"
-		docker pull crate:2.3
-		echo "Obtaining Latest Grafana"
-		docker pull grafana/grafana
-		echo "Obtaining Latest QuantumLeap"
-		docker pull smartsdk/quantumleap
-		echo "Obtaining Tutorial Application"
-		docker pull fiware/tutorials.context-provider
+		echo "Pulling Docker images"
```

Comment:
Agree that it's tidier. Also it prevents the issue of tags mismatching (note currently the create is pulling images not being used).
Docker is already echoing the name of each of the images being pulled, so that's already solved :)